simulation-runner/src/main/scala/com/puppetlabs/gatling/runner/ConfigDrivenSimulation.scala

```diff
@@ -101,7 +101,10 @@ class ConfigDrivenSimulation extends Simulation {
     val chainWithSleeps:ChainBuilder =
       addSleeps(chainWithFailFast, sleepDuration, numRepetitions)
 
+    val feeder = NodeFeeder("node", numInstances).circular
```

Comment:
ah. found it. https://github.com/puppetlabs/gatling-puppet-load-test/blob/34f30e164580933d6f216d56f878ed2fb193c987/simulation-runner/src/main/scala/com/puppetlabs/gatling/config/PuppetGatlingConfig.scala#L23